lms/templates/help_modal.html

```diff
@@ -19,196 +19,231 @@
-        strong_start=HTML('<strong>'),
-        strong_end=HTML('</strong>'),
-        link_start=HTML('<a href="{url}" target="_blank">').format(
-          url=marketing_link('FAQ')
-        ),
-        link_end=HTML('</a>'),
-        platform_name=static.get_platform_name())}
-      </p>
+        <p>${Text(_('Have a {strong_start}question about something specific{strong_end}? You can contact the {platform_name} general support team directly:')).format(
+            strong_start=HTML('<strong>'),
+            strong_end=HTML('</strong>'),
+            platform_name=static.get_platform_name()
+          )}</p>
+        <hr>
 
-    <p>${Text(_('Have a {strong_start}question about something specific{strong_end}? You can contact the {platform_name} general support team directly:')).format(
-        strong_start=HTML('<strong>'),
-        strong_end=HTML('</strong>'),
-        platform_name=static.get_platform_name()
-      )}</p>
-    <hr>
+        <div class="help-buttons">
+          <a href="#" id="feedback_link_problem">${_('Report a problem')}</a>
+          <a href="#" id="feedback_link_suggestion">${_('Make a suggestion')}</a>
+          <a href="#" id="feedback_link_question">${_('Ask a question')}</a>
+        </div>
 
-    <div class="help-buttons">
-      <a href="#" id="feedback_link_problem">${_('Report a problem')}</a>
-      <a href="#" id="feedback_link_suggestion">${_('Make a suggestion')}</a>
-      <a href="#" id="feedback_link_question">${_('Ask a question')}</a>
+        <p class="note">${_('Please note: The {platform_name} support team is English speaking. While we will do our best to address your inquiry in any language, our responses will be in English.').format(
+            platform_name=static.get_platform_name()
+          )}</p>
     </div>
 
-    <p class="note">${_('Please note: The {platform_name} support team is English speaking. While we will do our best to address your inquiry in any language, our responses will be in English.').format(
-        platform_name=static.get_platform_name()
-      )}</p>
+    <div id="feedback_form_wrapper">
 
-  </div>
+        <header></header>
 
-  <div class="inner-wrapper" id="feedback_form_wrapper">
-    <button class="close-modal">
-      <span class="icon fa fa-remove" aria-hidden="true"></span>
-      <span class="sr">
-        ## Translators: this is a control to allow users to exit out of this modal interface (a menu or piece of UI that takes the full focus of the screen)
-        ${_('Close')}
-      </span>
-    </button>
+        <form id="feedback_form" class="feedback_form" method="post" data-remote="true" action="/submit_feedback">
+          <div id="feedback_error" class="modal-form-error" aria-live="polite" tabindex="-1"></div>
+    % if not user.is_authenticated():
+          <label data-field="name" for="feedback_form_name">${_('Name')}*</label>
+          <input name="name" type="text" id="feedback_form_name" aria-required="true">
```

Comment:
Can we change the ```aria-required```  to ```required``` ? ```aria-required``` means its just required for screen reader users, while ```required``` means its required for everyone. Applies to the rest of the inputs that follow.